pyomo/pysp/tests/rapper/rapper_tester.py

```diff
@@ -110,6 +110,15 @@ def test_ef_solve_with_gap(self):
                                 tree_model = self.farmer_concrete_tree)
         res, gap = stsolver.solve_ef(solvername, tee=True, need_gap=True)
 
+    def test_ef_cvar_construct(self):
+        """ construct the ef with cvar """
+        stsolver = rapper.StochSolver("ReferenceModel.py",
+                                      fsfct = "pysp_instance_creation_callback",
+                                      tree_model = self.farmer_concrete_tree)
+        ef = stsolver.make_ef(generate_weighted_cvar = True,
+                              cvar_weight = 0.1,
+                              risk_alpha = 0.9)
```

Comment:
Minor suggestion would be to add a few assert statements to this test checking that the parameters are set correctly in the returned ef instance.